src/components/Form/UploadDocumentForm.jsx

```diff
@@ -84,14 +103,14 @@ const UploadDocumentForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleDocumentUpload} autoComplete="off">
+      <form onSubmit={handleDocUpload} autoComplete="off">
         <label htmlFor="verify-checkbox">
           Verify File on upload:
           <input
             id="verify-checkbox"
             type="checkbox"
-            value={state.verifyFile}
-            onClick={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
+            value={verifyFile}
+            onChange={(e) => setVerifyFile(e.target.value)}
```

Comment:
The setVerifyFile seems to just set the initial state of false for verifyFile since it is referencing the value in line 112.

https://github.com/codeforpdx/PASS/assets/14917816/f1aff375-de6d-483d-a811-ea2b558810a6

Would need to set the onChange to `() => setVerifyFile(!verifyFile)` instead. Think we could use the condition for `checked={verifyFile}` to toggle the state with onChange.

https://github.com/codeforpdx/PASS/assets/14917816/b0f65807-20e8-49f7-a0bb-86ad0c8fc17f

Reply:
fixed